R/ttestindependentsamples.R

```diff
@@ -190,8 +190,14 @@ TTestIndependentSamples <- function(jaspResults, dataset = NULL, options, ...) {
   if (!options$equalityOfVariancesTests || !is.null(container[["equalityVariance"]]))
     return()
 
+  if (options$equalityOfVarianceType == "brownForsythe")
+    nameOfEqVarTest <- gettext("Brown-Forsythe")
+  else if (options$equalityOfVarianceType == "levene")
+    nameOfEqVarTest <- gettext("Levene's")
```

Comment:
```suggestion
  nameOfEqVarTest <- switch(
    options$equalityOfVarianceType,
    "brownForsythe" = gettext("Brown-Forsythe"),
    "levene" = gettext("Levene's")
  )
```

`switch()` is slightly more readable, but it's up to you whether you want to change it, `if` `else` is not wrong.